stage-0-getting-started.md

```diff
@@ -48,5 +48,12 @@ Here are some starting points for learning about the Ethereum protocol.
    - Stateless Ethereum: https://blog.ethereum.org/2019/12/30/eth1x-files-state-of-stateless-ethereum/
    - Kate Commitments: https://hackmd.io/yqfI6OPlRZizv9yPaD-8IQ?view#Background
 
+#### Sharding
```

Comment:
I think that it's very possible these links are out of date and I don't want to publish them as relevant without being sure they are up-to-date.  Here's one that is definitely up-to-date: https://vitalik.ca/general/2021/04/07/sharding.html